.github/workflows/ci.yml

```diff
@@ -24,7 +24,31 @@ env:
   LOCALE_ID: 'en-US'
 
 jobs:
+  check-run:
```

Comment:
@lognaturel What do you think? 😁

Reply:
Yup, that looks great to me!

I was trying to think about why we don't have this issue for other subteams and I realized that it's because typically even core devs tend to work on branches on their forks rather than directly on the repo. I think that means there still are duplicate CI runs but they're not showing up in the same place.